script/set-node-version

```diff
@@ -0,0 +1,12 @@
+nvm_available() {
```

Comment:
It looks like this scripts is missing a `Shebang`. I think it also has incorrect permissions.

Reply:
It's missing a shebang b/c it's not designed to be directly executed as a script.

I don't think we need executable permissions here, because the file itself is sourced by the shell via another script, not executed.

Reply:
I think there's definitely an argument to add .sh as a suffix to this (and other scripts in this folder) to make it clearer what the files contain, mind!

Reply:
Possibly we could use `#!/bin/false`?

Reply:
Ooh, just googled this – perfect! For others who are unsure, according to [the internet](https://stackoverflow.com/a/41849445/3631858), #!/bin/false is a safety mechanism of preventing execution of include-only files.